### ISSUE TITLE
Linux: does not detect amd radeon 7900XTX

### ISSUE THREAD

**Issue by aamaanaa** [Pavelevich/llm-checker#9]
on linux it is not able to find amd gpu 7900xtx with 24GB vram:

```bash
=== Hardware Detection ===

Summary:

AMD Ryzen 9 9950X3D 16-Core Processor (31GB RAM, CPU-only)

Tier: MEDIUM HIGH

Max model size: 20GB

Best backend: cpu

CPU:

AMD Ryzen 9 9950X3D 16-Core Processor

Cores: 32 (16 physical)

SIMD: AVX512

[OK] AVX-512

[OK] AVX2

Fingerprint: cpu-amd-ryzen-9-9950x3d-16-core--32c-avx512
```

i have tested out some 30b models wich i can run.

if you need more info please do ask

method of running was:

```bash
npx llm-checker hw-detect
```

**Comment by aamaanaa** [comment]
yes that fixed it. Thanks.
